sys/zuse.hoon

```diff
@@ -7942,4 +7936,148 @@
       --
     --
   --
+:: |dawn: pre-boot request/response de/serialization and validation
```

Comment:
Is `|something` new comment syntax for a core arm? Are single spaces for comments considered good style these days? How far have we fallen? (Also still strongly not a fan of the whitespace between the comment and the thing, combined with no whitespace between the previous thing and the comment, but maybe that's a losing battle.)

Reply:
I believe this is proper space-rated style. @belisarius222, can you confirm?

Reply:
Yes, `|arm-name` for arms that define cores that aren't gates. And yes, space-rated style has comment, then blank comment line, then target of the comment, with no blank line above the comment.

There should be two spaces after the colcol, however.

Reply:
> space-rated style has comment, then blank comment line, then target of the comment, with no blank line above the comment.

What's the reasoning for doing a blank line between the comment and the thing it's about, and/but not between the thing it's not about and the comment? I brought this up when this style first surfaced, I think it's not particularly pleasant to read.

Reply:
@belisarius222, I don't know how I got the single-space stuck in my head. This might be wrong in `:acme` and it's libs too. Two is definitely better.